ONSdigital/sds
# Sdss 848 dataset happy path

### Motivation and Context

Separates dataset integration test focusing on the happy path to simplify the test. 

### What has changed



- Split integration test for dataset endpoints (happy paths) into its own test class file (DatasetEndpointsIntegrationTest)
- Added tests for uploading and retrieving dataset metadata, fetching unit data, and handling datasets with missing titles.
- Introduced firestore helper function to upload datasets to firestore

### How to test?



Run the integration tests  check if datasets are uploaded with correct unit data and metadata.


### Links



https://jira.ons.gov.uk/browse/SDSS-848

### Screenshots (if appropriate):
